commit b3ef5627c6339dd09a9a352e9b5960f6fd829159 (refs/changes/10/3310/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-09T11:34:27-08:00 (1 year, 8 months ago)
    
    TOOLS-1981 avoid unexpected changes to package.json in Joyent repos
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/tools/mk/Makefile.deps b/tools/mk/Makefile.deps
index 7419253..91f8346 100644
--- a/tools/mk/Makefile.deps
+++ b/tools/mk/Makefile.deps
@@ -66,14 +66,14 @@ $(ESLINT_EXEC): package.json | $(NPM_EXEC)
 	ESLINT_VER=$$($(NODE) -e 'console.log(require("./package.json").devDependencies["eslint"] || "")') && \
 	    ESLINT_JOY_VER=$$($(NODE) -e 'console.log(require("./package.json").devDependencies["eslint-plugin-joyent"] || "")') && \
 	    [[ -n $$ESLINT_VER && -n $$ESLINT_JOY_VER ]] && \
-	    $(NPM) install eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
+	    $(NPM) install --no-save eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
 	    touch $(ESLINT_EXEC)
 else
 $(ESLINT_EXEC): package.json
 	ESLINT_VER=$$(node -e 'console.log(require("./package.json").devDependencies["eslint"] || "")') && \
 	    ESLINT_JOY_VER=$$(node -e 'console.log(require("./package.json").devDependencies["eslint-plugin-joyent"] || "")') && \
 	    [[ -n $$ESLINT_VER && -n $$ESLINT_JOY_VER ]] && \
-	    npm install eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
+	    npm install --no-save eslint@$$ESLINT_VER eslint-plugin-joyent@$$ESLINT_JOY_VER && \
 	    touch $(ESLINT_EXEC)
 endif
 
